Bluetooth/nRF51822/PhoneGap/BLE-Buzzy-Pixel/www/js/status-service.js

```diff
@@ -0,0 +1,107 @@
+var intervalID;
+
+function objectValues(obj) {
+  return Object.keys(obj).map(key => obj[key]);
+}
+
+function startPolling() {
+  stopPolling();
+  intervalID = setInterval(requestStatus, 10000);
+}
+
+function stopPolling() {
+  if (intervalID) {
+    clearInterval(intervalID);
+  }
+  intervalID = null;
+}
+
+function togglePolling(btn) {
+  if (intervalID) {
+    stopPolling();
+    if(btn) {
+      btn.value = "Start";
+    }
+  } else {
+    startPolling();
+    if (btn) {
+      btn.value = "Stop";
+    }
+  }
+}
+
+function postNewStatus(url, newStatusValue) {
+  cordovaFetch(url, {
+    method: 'POST',
```

Comment:
As discussed, will see if plugin can be fixed to handle Put, if not try some polyfill for fetch using XHR